### PR TITLE
fix(#80): Different names for Washington's Birthday

### LINKED ISSUE #80: in MA Washington's b-day -> Presidents Day
According to wikipedia it depends largely on state:
https://en.wikipedia.org/wiki/Washington%27s_Birthday

### PR DESCRIPTION
**Pull request by commenthol**
Tries to fix #80. 
Different names per state for Predident's Day / Washington's Birthday

### CODE REVIEW

File: data/countries/US.yaml [hunk 1]
```diff
@@ -703,6 +778,9 @@ holidays:
           02-15:
             name:
               en: Susan B. Anthony Day
+          3rd monday in February:
+            name:
+              en: President's Day
```
Comment (gerryster) [inline data/countries/US.yaml]: I am not seeing that Wisconsin has a special name in our sources. I think this section can be removed.

File: data/countries/US.yaml [hunk 2]
```diff
@@ -44,10 +45,10 @@ holidays:
         _name: 01-01
       3rd monday in January:
         name:
-          en: Martin Luther King Day
+          en: Martin Luther King Jr. Day
       3rd monday in February:
         name:
-          en: Washington’s Birthday
+          en: Presidents' Day
```
[the remark this answers is not in the document]
Comment (gerryster) [reply data/countries/US.yaml]: Given my suggestion, I think that Hawaii, New Mexico, North Dakota, Oklahoma, Pennsylvania, South Dakota, Texas, Vermont, and Washington state would need to be updated to "Presidents' Day."

File: data/countries/GU.yaml [hunk 3]
```diff
@@ -14,6 +16,7 @@ holidays:
       - US
     # additional holidays
     days:
+      3rd monday in February: false
```
[the remark this answers is not in the document]
Comment (gerryster) [reply data/countries/GU.yaml]: You can disregard this. After looking deeper into holidays in Guam, I agree with your change...President's Day does not appear to be celebrated here.

File: data/countries/US.yaml [hunk 4]
```diff
@@ -44,10 +45,10 @@ holidays:
         _name: 01-01
       3rd monday in January:
         name:
-          en: Martin Luther King Day
+          en: Martin Luther King Jr. Day
```
[the remark this answers is not in the document]
Comment (commenthol) [reply data/countries/US.yaml]: https://en.wikipedia.org/wiki/Federal_holidays_in_the_United_States#Martin_Luther_King_Jr._Day states it as "Martin Luther King Jr. Day". So what shall I use?

File: data/countries/US.yaml [hunk 5]
```diff
@@ -148,6 +151,9 @@ holidays:
           3rd monday in January:
             name:
               en: Dr. Martin Luther King Jr./ Civil Rights Day
+          3rd monday in February:
+            name:
+              en: Lincoln/ Washington/ Presidents' Day
```
[the remark this answers is not in the document]
Comment (commenthol) [reply data/countries/US.yaml]: Will change this.

**Comment by commenthol** [comment]
Hi I tried my best to solve the names. Could you please review again?

File: data/countries/US.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (gerryster) [reply data/countries/US.yaml]: Ok, I've done some digging, and I would stick with "Martin Luther King Jr. Day". 

https://en.wikipedia.org/wiki/Martin_Luther_King_Jr._Day cites the "birthday of" version as the official name. However, the reference is https://www.opm.gov/policy-data-oversight/snow-dismissal-procedures/federal-holidays/#url=2019 . When I found the actual legislation which created the holiday (https://www.govinfo.gov/content/pkg/STATUTE-98/pdf/STATUTE-98-Pg1473.pdf), it does not define any official name. I think "Martin Luther King Jr. Day" sounds more natural and that the Office of Public Management is being overly formal.

File: data/countries/US.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (gerryster) [reply data/countries/US.yaml]: This all looks good except that Hawaii is a President's Day state.

File: data/countries/US.yaml [hunk 6]
```diff
@@ -431,6 +468,9 @@ holidays:
         zones:
           - America/Chicago
         days:
+          3rd monday in February:
+            name:
+              en: Lincoln's and Washington's Birthday
```
[the remark this answers is not in the document]
Comment (gerryster) [reply data/countries/US.yaml]: Do you want to add Lincoln't Birthday to Missouri in this PR?

**Comment by gerryster** [comment]
Ok, this is looking good. Just one overridden name needed in Hawaii and an optional suggestion for Missouri while we are thinking of presidential birthday holidays.

Also, could I talk you into adding new commits instead of force pushes for pull request branches? It makes things a bit disorienting as the reviewer. Also, I often times pull a local copy of a branch for a non-trivial PR as I prefer to navigate through the change in my editor and a force push means that I need to delete my local branch before pulling.

**Comment by commenthol** [comment]
Hi @gerryster can you please review the changes made on US-HI and US-MO? Thanks. 
I will consider your point on force-pulls - thanks for the feedback.

**Comment by gerryster** [comment]
Looks good! 

It looks like Rravis is having some trouble installing one of the dependencies.

**Comment by commenthol** [comment]
travis is fixed on master. There were some changes related to Typescript testing.

**Pull request merged**